src/mscorlib/model.xml

```diff
@@ -5915,6 +5915,8 @@
       <Member Name="PrelinkAll(System.Type)" />
       <Member Name="PtrToStringAnsi(System.IntPtr)" />
       <Member Name="PtrToStringAnsi(System.IntPtr,System.Int32)" />
+      <Member Name="PtrToStringUTF8(System.IntPtr)" />
```

Comment:
Please add these to System.Private.CoreLib model.xml as well.
cc @gkhanna79 @AlexGhiondea